core/server/master/src/main/java/alluxio/master/block/DefaultBlockMaster.java

```diff
@@ -708,12 +709,24 @@ private void processWorkerAddedBlocks(MasterWorkerInfo workerInfo,
             mLostBlocks.remove(blockId);
           }
         } else {
-          LOG.warn("Failed to register workerId: {} to blockId: {}", workerInfo.getId(), blockId);
+          LOG.warn("Invalid block: {} from worker {}.", blockId,
+              workerInfo.getWorkerAddress().getHost());
         }
       }
     }
   }
 
+  @GuardedBy("workerInfo")
+  private void processWorkerOrphanedBlocks(MasterWorkerInfo workerInfo) {
+    for (long block: workerInfo.getBlocks()) {
+      if (!mBlocks.containsKey(block)) {
+        LOG.warn("Requesting delete for orphaned block: {} from worker {}.", block,
```

Comment:
I'd prefer `info` here; we already warn in `processWorkerAddedBlocks`, and the message sounds more informational than warn-like

Reply:
Fixed, thanks.